dll/cpl/access/lang/cs-CZ.rc

```diff
@@ -282,10 +290,10 @@ STRINGTABLE
 BEGIN
     IDS_CPLSYSTEMNAME "Možnosti usnadnění"
     IDS_CPLSYSTEMDESCRIPTION "Upraví možnosti usnadnění pro tento počítač."
-    IDS_SENTRY_NONE "[None]"
-    IDS_SENTRY_TITLE "Flash the titlebar"
-    IDS_SENTRY_WINDOW "Flash the active window"
-    IDS_SENTRY_DISPLAY "Flash the desktop"
-    IDS_SECONDS "Seconds"
-    IDS_MINUTES "Minutes"
+    IDS_SENTRY_NONE "[Žádné]"
+    IDS_SENTRY_TITLE "Zablikat horní lištou"
+    IDS_SENTRY_WINDOW "Zablikat aktivním oknem"
+    IDS_SENTRY_DISPLAY "Zablikat plochou"
+    IDS_SECONDS "sekund"
+    IDS_MINUTES "minut"
```

Comment:
Just to be sure: in czech there's no capital letter for "sekund" or "minut", when these are displayed as follows:
"12.34 sekund"   and   "10 minut" ?

Reply:
Yes, there is no capital letter in those cases.